pyproject.toml

```diff
@@ -21,7 +21,8 @@ readme = "README-wheel.md"
 authors = [
   {name="PyTorch Team", email="packages@pytorch.org"},
 ]
-license = {file = "LICENSE"}
+license = "BSD-2-Clause"
```

Comment:
We're actually BSD-3-Clause 

https://github.com/pytorch/executorch/blob/main/LICENSE

The last clause there makes it 3-clause BSD

Reply:
thank you for the correction, updated